### PR TITLE
Updated config to use new template and master branch

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Kicking off CI again.

**Comment by Blackbaud-BobbyEarl** [comment]
@Blackbaud-SteveBrush I know you already approved this, but I didn't feel right merging it as I made code changes after your review.  I also updated the branch settings to dismiss stale reviews so that I don't have to make that decision in the future.  :-)

File: e2e/skyux-lib-help-tests/skyux-lib-help.e2e-spec.js [hunk 1]
```diff
@@ -102,11 +83,12 @@ function addModalToHomePage() {
 
 describe('skyux lib help', () => {
   beforeAll((done) => {
-    prepareBuild()
-    .then(() => {
-      done();
-    });
-    addModalToHomePage();
+    common.exec('npm', ['i', '--no-save', '@skyux/modals'])
```
Comment (Blackbaud-SteveBrush) [inline e2e/skyux-lib-help-tests/skyux-lib-help.e2e-spec.js]: Are builder's `devDependencies` not recognized for this? https://github.com/blackbaud/skyux-sdk-builder/blob/master/package.json#L141

Comment (Blackbaud-BobbyEarl) [reply e2e/skyux-lib-help-tests/skyux-lib-help.e2e-spec.js]: Ah good catch.  This is an artifact of me debugging the failure.  Reverting now.

**Pull request merged**